pkg/plugins/golang/v3/scaffolds/internal/templates/main.go

```diff
@@ -196,7 +198,6 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"
```

Comment:
@alvaroaleman @joelanford  do you remember why we implement the log zap feature in controller-runtime instead of using klog? 

c/c @jmrodri @varshaprasad96

Reply:
/hold

Only to avoid it getting merged before it has enough reviews and discussions.